### PR TITLE
fixed cli install for esm and typescript

### PR DESCRIPTION
**Pull request by AlexGalichenko**

### CODE REVIEW

File: src/install.ts [hunk 1]
```diff
@@ -146,4 +156,11 @@ export default async function install(): Promise<void> {
         cwd: process.cwd(),
         respectNpm5: true
     });
+
+    console.log('test script:');
```
Comment (nomid0612) [inline src/install.ts:160]: do we need console.log here?

Comment (AlexGalichenko) [reply src/install.ts:160]: yes, it informs user how to run qava tests depends on module system selected

Comment (ALegchilov) [reply src/install.ts:160]: Could it be better with winston logger?

Comment (AlexGalichenko) [reply src/install.ts:160]: I don't think so, it's not a log, it's console message for the user

Comment (ALegchilov) [reply src/install.ts:160]: `colsole.log` looks like debugging snippet occasionly left by a developer, whereas `console.info` looks rather neutral, though both provide the same functionality. It's just my opinion, not blocking the pull request.

Comment (AlexGalichenko) [reply src/install.ts:160]: anyway I think it's should not be a blocker for current PR

**Review by ALegchilov — APPROVED** [review]

**Pull request merged**